overlord/snapstate/snapmgr.go

```diff
@@ -560,6 +660,12 @@ func (m *SnapManager) Ensure() error {
 
 	m.runner.Ensure()
 
+	// needs to run after the runner
```

Comment:
_Why_ it needs to run after the runner? Otherwise it's a riddle for the next reader.